.gitignore

```diff
@@ -0,0 +1,33 @@
+# Logs
```

Comment:
please don't include this in this PR - keep a global gitignore if your editors add these files.